File: JS/10-18-25/JS/LCM.js
let number1 = 6;
let number2 = 15;
let LCM = 1;
let multiple = 2;

while (number1 != 1 || number2 != 1){
	if( number1 % multiple == 0 || number2 % multiple == 0){
		if(number1 % multiple == 0)
			number1 /= multiple;
		if (number2 % multiple == 0)
			number_2 /= multiple;
		LCM *= multiple;}
	else
		multiple++;
	}

File: JS/10-18-25/Python/LCM.py
number_1 = 6
number_2 = 15
LCM = 1
multiple = 2

while number_1 != 1 or number != 1:
	if number_1 % multiple == 0 or number_2 % multiple == 0:
		if number_1 % multiple == 0:
			number_1 //= multiple
		if number_2 % multiple == 0:
			number_2 //= multiplw
		LCM *= multiple

	else:
		multiple += 1
print(f"The LCM is {LCM})